### ISSUE TITLE
cycleStatus.cycleRunning == 3?

### ISSUE THREAD

**Issue by Hixie** [GEMakers/gea-plugin-dishwasher#13]
GE GDF570SGFWW dishwasher. Heavy cycle. operatingMode is End of Cycle.

When I query cycleState, I get cycleRunning:3. The documentation says that cycleRunning should be either 0 to indicate that it's not running, or non-zero if it is. It's not running as far as I can tell.

Normally I get 1 when it's running.

**Comment by Hixie** [comment]
looks like it's the selected program. 0 means inactive, 1 means autosense, 3 means heavy, 6 means normal, and 11 means light.

**Comment by palisaide** [comment]
@Hixie: You're right, `cycleStatus.cycleRunning` represents the enumerated value of the cycle that is currently running.  The documentation is wrong and needs to be updated.
